### DISCUSSION TITLE
How come my Github account and github.io do not show up on Google search?

### DISCUSSION THREAD

**Discussion by MadisonButchko** [community/community#139222] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi, I have a question as to why my website and github account do not show up when I Google it? Why it is not found on google? Can someone please help here and inform me how to get it to show up on Google Search? It is just because it is a new site that I created last week or...?

**Comment by davevad93** [comment]
Hi @MadisonButchko ,

It's probably because your account and the `github.io` page haven't been indexed yet by the Google search engine, in my case I had to wait a while for my username to appear on Google, the same happens if you change your username; when I changed mine, I had to wait almost 1 month for it to appear in the search results.

For more context, see the answer in this discussion #44421.

**Comment by MrKuip** [comment]
I have the same issue. Not a trace of my project in google search. I searched with duck duck go and I was in second place.

**Reply by alperenproo** [reply]
how did you fix it ?

**Comment by MrKuip** [comment]
No fix

-------- Original Message --------
